meta-mender-qemu/recipes-extended/sudo/sudo_%.bbappend

```diff
@@ -0,0 +1,13 @@
+FILES_${PN} += "/data/usr"
+do_install_append_qemux86-64 () {
+ mkdir -p ${D}/data/usr/bin
+ mv ${D}/usr/bin/sudo ${D}/data/usr/bin/
+ ln -s /data/usr/bin/sudo ${D}/usr/bin/sudo
```

Comment:
Standard indentation in meta-mender is four spaces.

Reply:
right! thanks!